cmd/gitbase/command/server.go

```diff
@@ -66,12 +79,54 @@ func (c *Server) Execute(args []string) error {
 		{Password: c.Password},
 	}
 
+	var tracer opentracing.Tracer
+	if c.TraceEnabled {
+		if os.Getenv("JAEGER_SERVICE_NAME") == "" {
```

Comment:
Maybe instead of checking an environment variable, will be less error prone check the content of https://sourcegraph.com/github.com/src-d/gitbase@8844a396d235d32143f77bf85ec9a1820bbf538d/-/blob/vendor/github.com/uber/jaeger-client-go/config/config.go#L38 into the cfg object.

Reply:
The configuration is still not created at that point. Also, setting the environment is what fills `ServiceName` in `FromEnv`. That's how they do in the examples:

https://github.com/jaegertracing/jaeger-client-go/blob/master/config/example_test.go#L107-L110

Reply:
Changed :+1: